src/components/ResourceForm.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+export function ResourceForm() {
+  const [input, setInput] = useState('');
```

Comment:
we really should have types on these and the components, but we can save that for a future PR when the style checking is added.